### PR TITLE
Added bootstrap

### PR DESCRIPTION
**Pull request by RichardLitt**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -83,24 +83,139 @@ TODO
 
 
 # Group bootstrap
+Show or edit the list of bootstrap peers.
+
+Running 'ipfs bootstrap' with no arguments will run 'ipfs bootstrap list'.
+
+SECURITY WARNING:
```
Comment (dignifiedquire) [inline apiary.apib]: Lower case and markdown formatting like bold please

File: apiary.apib [hunk 2]
```diff
@@ -83,24 +83,139 @@ TODO
...
+
+The bootstrap command manipulates the "bootstrap list", which contains
```
Comment (dignifiedquire) [inline apiary.apib]: Instead of quotes use inline code ticks

File: apiary.apib [hunk 3]
```diff
@@ -83,24 +83,139 @@ TODO
...
 
 ## add [GET /bootstrap/add]
+Add peers to the bootstrap list
+
+Outputs a list of peers that were added (that weren't already
+in the bootstrap list).
+
++ Parameters
+    + arg (Multiaddr, required) - A peer to add to the bootstrap list
+    + default (boolean, optional) - add default bootstrap nodes
```
Comment (dignifiedquire) [inline apiary.apib:105]: You need add these to the query above, like this `[GET /bootstrap/add{?arg,&default}]` See https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md#examples for details

File: apiary.apib [hunk 4]
```diff
@@ -83,24 +83,139 @@ TODO
...
+
+        ```
+        {
+          "Peers": [
+            "/ip4/5.9.33.222/tcp/4001/ipfs/QmNRV7kyUxYaQ4KzxFXPYm8EfuzJbtGn1wSFenjXL6LD8y"
+          ]
+        }
+        ```
+
++ Response 400 (text/json)
```
Comment (Dignifiedquire) [inline apiary.apib]: 400?

Comment (dignifiedquire) [inline apiary.apib]: also not `text/json` but rather `application/json`

File: apiary.apib [hunk 5]
```diff
@@ -83,24 +83,139 @@ TODO
...
+    + Body
+
+        ```
+        {
+          "Message": "no bootstrap peers to add",
+          "Code": 1
+        }
+        ```
+
++ Response 500 (text/json)
```
Comment (Dignifiedquire) [inline apiary.apib]: 500?

File: apiary.apib [hunk 6]
```diff
@@ -83,24 +83,139 @@ TODO
...
 + Response 200 (text/plain)
 
+    + Attributes (object)
+        + Peers (array)
+            - (Multihash)
+    + Body
+
+        ```
+        {
+        "Peers": [
```
Comment (dignifiedquire) [inline apiary.apib]: Indent please

File: apiary.apib [hunk 7]
```diff
@@ -83,24 +83,139 @@ TODO
...
+        }
+        ```
+
++ Response 500 (text/json)
+
+    + Attributes (object)
+        + Message: invalid ipfs address (string)
+        + Code: 0 (number)
+
+    + Body
```
Comment (dignifiedquire) [inline apiary.apib:211]: No need to specify the body if you specify examples for all attributes

File: apiary.apib (hunk 4 above)
Comment (RichardLitt) [inline apiary.apib]: Changed, but I'm not sure how to tell the difference. Is `text/plain` also a valid mimetype for some responses?

File: apiary.apib (hunk 7 above)
Comment (RichardLitt) [inline apiary.apib:211]: Don't I specify examples of all attributes for the other methods, too? At what point is the body not redundant?

Comment (dignifiedquire) [inline apiary.apib:211]: If you specify examples it's not redundant only for arrays, I think.

File: apiary.apib (hunk 4 above)
Comment (dignifiedquire) [inline apiary.apib]: yes, if you use postman it should show you the mime type in the headers

**Comment by dignifiedquire** [comment]
Thanks :octocat:

**Pull request merged**